resources/assets/components/CampaignSingle/index.js

```diff
@@ -134,7 +135,7 @@ class CampaignSingle extends React.Component {
 
     return (
       <div className="container">
-        <StatusCounter postTotals={this.props.post_totals} campaign={campaign} />
+        <StatusCounter postTotals={this.props.post_totals} campaign={campaign} download={downloads}/>
```

Comment:
I don't think the download button should be coupled to the `StatusCounter` component. It probably should be it's own component so it can be displayed separated from the `StatusCounter` whenever we need it.